apps/web/app/sentry/page.tsx

```
@@ -0,0 +1,54 @@
+  if (shouldError) {
+    triggerError();
+  }
+
+  return (
+    <div className="p-8">
+      <h1>Sentry Sourcemap Test</h1>
+      <div className="space-y-4">
+        <button 
+          onClick={() => setShouldError(true)}
+          className="bg-red-500 text-white px-4 py-2 rounded"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)